### PR TITLE
Release v0.23.0

### PR DESCRIPTION
**Pull request by pjbgf**

### CODE REVIEW

**Review by hiddeco — APPROVED** [review]

**Comment by stefanprodan** [comment]
@hiddeco maybe wait for #607 to be merge first.

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,44 @@
...
+decryption operations.
+
+**Breaking changes**:
+
+- Use of file-based KubeConfig options are now permanently disabled (e.g. 
+`TLSClientConfig.CAFile`, `TLSClientConfig.KeyFile`, `TLSClientConfig.CertFile`
+and `BearerTokenFile`). The drive behind the change was to discourage
+insecure practices of storing crypto material in disk, users should instead
+use the in-memory alternatives instead and follow best practices around
+encrypting and keeping such information protected.
```
Comment (stefanprodan) [inline CHANGELOG.md:25]:
```suggestion
and `BearerTokenFile`). The drive behind the change was to discourage
insecure practices of mounting Kubernetes tokens inside the controller's container file system.
```

Comment (pjbgf) [reply CHANGELOG.md]: Applied the changes at both KC and HC PRs. PTAL

**Review by stefanprodan — APPROVED** [review]
🚀

File: CHANGELOG.md (hunk 1 above)
Comment (stefanprodan) [reply CHANGELOG.md]: thanks!

**Pull request merged**